sql/src/main/java/io/crate/analyze/relations/JoinPair.java

```diff
@@ -95,6 +95,10 @@ boolean equalsNames(QualifiedName left, QualifiedName right) {
         return this.left.equals(left) && this.right.equals(right);
     }
 
+    boolean equalsName(QualifiedName relation) {
```

Comment:
maybe rename this to contains? I think otherwise it might be a bit confusing as there is also a `equalsNames`